virtualizarr/tests/test_manifests/test_store.py

```diff
@@ -16,14 +16,11 @@
     ManifestGroup,
     ManifestStore,
 )
-from virtualizarr.tests import (
-    requires_obstore,
-)
+from virtualizarr.tests import requires_obstore
 
 
 @pytest.fixture()
-@requires_obstore
```

Comment:
Why does this test not require obstore?

Reply:
These markers don't make any impact on fixtures, so I removed it. The fixture is generated if any tests using the fixtures are captured by pytest, so the markers should all go on tests rather than fixtures.